utils/testUserAuth.js

```diff
@@ -1,11 +1,12 @@
 const User = require('~/models/user')
 const { hashPassword } = require('~/utils/passwordHelper')
 
-const testUserAuthentication = async (app, testUser = {}) => {
+const testUserAuthentication = async (app, userData = {}) => {
```

Comment:
you changed testUserAuthentication without checking maybe other tests pass testUser to it

Reply:
logic that used in other cases are remain the same